crates/storage/src/lazy/cache_cell.rs

```diff
@@ -36,11 +36,6 @@ impl<T> CacheCell<T> {
             inner: UnsafeCell::new(value),
         }
     }
-
-    /// Returns the inner value.
-    pub fn into_inner(self) -> T {
-        self.inner.into_inner()
-    }
```

Comment:
Hm, ... so I guess this is dead code now after these changes. However, I am not sure if removing this actually makes sense since `CacheCell` is a low-level API intended for general use, not only ink! internal use.

Reply:
Brought it back.